managed/CONTRIBUTING.md

```diff
@@ -53,25 +53,25 @@ go test -timeout=30s -p 1 ./...
-| PERCONA_TEST_NICER_API                     | Enables nicer API with default/zero values in response.                                                             | false                                    |
-| PERCONA_TEST_VERSION_SERVICE_URL           | Sets versions service URL                                                                                           | https://check.percona.com/versions/v1    |
-| PERCONA_TEST_CHECKS_FILE                   | Specifies path to local checks file and disables downlading checks files from Percona Platform                      | none                                     |
-| PERCONA_TEST_CHECKS_DISABLE_START_DELAY    | Disables checks service startup delay                                                                               | false                                    |
-| PERCONA_TEST_TELEMETRY_INTERVAL            | Sets telemetry reporting interval                                                                                   | 24h                                      |
-| PERCONA_TEST_TELEMETRY_DISABLE_SEND        | Disables sending of telemetry data to SaaS. This param doesn't affect telemetry data gathering from the datasources | false                                    |
-| PERCONA_TEST_TELEMETRY_FILE                | Sets path for telemetry config file                                                                                 |                                          |
-| PERCONA_TEST_TELEMETRY_DISABLE_START_DELAY | Disable the default telemetry execution start delay, so that telemetry gathering is run immediately upon system     | false                                    |
-| PERCONA_TEST_TELEMETRY_RETRY_BACKOFF       | Sets telemetry reporting retry backoff time                                                                         | 1h                                       |
-| PERCONA_TEST_PLATFORM_ADDRESS              | Sets Percona Platform address                                                                                       | https://check.percona.com                |
-| PERCONA_TEST_PLATFORM_INSECURE             | Allows insecure TLS connections to Percona Platform                                                                 | false                                    |
-| PERCONA_TEST_PLATFORM_PUBLIC_KEY           | Sets Percona Platform public key (Minisign)                                                                         | set of keys embedded into managed binary |
+| Variable                                | Description                                                                                                         | Default                                  |
+|-----------------------------------------| ------------------------------------------------------------------------------------------------------------------- | ---------------------------------------- |
```

Comment:
I wonder if there's a canonical link we can reference here (instead of trying to list all possible variables), the closest I could find was https://docs.percona.com/percona-monitoring-and-management/setting-up/server/docker.html?h=enable_vm_cache#environment-variables, but it doesn't seem complete

Reply:
these ones is for development purpose, that's why it's the best place to keep